### PR TITLE
Disabling Paste in all user authenticaiton forms(#4487)

### LINKED ISSUE #4487: Disabling paste in /login form
A user is able to paste password from any normal text-field to the password field in all forms PublicLab uses for user authentication. This should be resolved since user should not be able to paste the password in the password field.
peek 2019-01-01 16-15

### PR DESCRIPTION
**Pull request by kunalvaswani123**
Fixes #4487  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] PR is descriptively titled 📑
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by kunalvaswani123** [comment]
@SidharthBansal I have created the issue and the pr linked to it.
Also here is the gif for verification:
disablingpaste

**Comment by kunalvaswani123** [comment]
I dont understand the error travis has given could you help please ?

**Comment by SidharthBansal** [comment]
Close and open the pr so that you can restart the build.
Changes looks good to me.

**Comment by SidharthBansal** [comment]
@gauravano @jywarren please review the tests. I think we need to update the key secrets again. Many prs are failing.
Happy New Year Everyone. I hope you all enjoyed a lot. Wishing you and your family very happy new year.

**Comment by gauravano** [comment]
Restarting the build

**Comment by kunalvaswani123** [comment]
@SidharthBansal I have done the changes in all the forms and also in _create_form partial I have changed 
the placeholder just like I did in our previous PR but not in _edit_form to avoid conflicts

**Comment by kunalvaswani123** [comment]
gif for _edit_form:
disablingpaste2
gif for signup partial:
disablingpaste3
gif for login modal:
disablingpaste4

File: app/views/users/_edit_form.html.erb [hunk 1]
```diff
@@ -41,7 +41,8 @@
         <%= f.password_field :password, { placeholder: I18n.t('users._form.password'),
                                           tabindex: 4,
                                           class: 'form-control',
-                                          id: 'password' }
+                                          id: 'password',
+                                          onpaste: 'return false;' }
```
Comment (SidharthBansal) [inline app/views/users/_edit_form.html.erb]: >>not in _edit_form to avoid conflicts

But you have done changes here.

**Comment by SidharthBansal** [comment]
@kunalvaswani123 well done.
Can you please give the gif for the /signup page also?
Thanks

File: app/views/users/_edit_form.html.erb (hunk 1 above)
Comment (kunalvaswani123) [reply app/views/users/_edit_form.html.erb]: not placeholder one

Comment (kunalvaswani123) [reply app/views/users/_edit_form.html.erb]: because I'll be doing that in this https://github.com/publiclab/plots2/pull/4386

**Comment by kunalvaswani123** [comment]
@SidharthBansal signup page uses the same partial as signup modal still I'll give you the gif

**Comment by kunalvaswani123** [comment]
@SidharthBansal here is the gif:
disablingpaste5

**Comment by SidharthBansal** [comment]
Thanks. I know both use same partial but just want to check it out before merging.
Thankyou Kunal.
Merging it up @jywarren

**Pull request merged**